docker/Dockerfile

```diff
@@ -0,0 +1,13 @@
+# Setup the base image
+FROM python:3.8-slim-bullseye
+
+# Activate venv
+RUN python3.8 -m venv /venv
+ENV PATH="venv/bin:$PATH"
+
+# Setup DeepSparse
+RUN pip3 install deepsparse[server]
+
+# Bash as default entrypoint
+CMD ["bin/bash"]
```

Comment:
should likely switch `CMD` to `ENTRYPOINT` to allow for override

Reply:
this is not necessary since the base image by default has python set as an entry point.